docs/src/main/sphinx/functions/json.rst

```diff
@@ -2,6 +2,1128 @@
 JSON functions and operators
 ============================
 
+The SQL standard describes functions and operators to process JSON data. They
+allow you to access JSON data according to its structure, generate JSON data,
+and store it persistently in SQL tables.
+
+Importantly, the SQL standard imposes that there be no dedicated data type to
+represent JSON data in SQL. Instead, JSON data shall be represented as
+character or binary strings. Although Trino supports ``JSON`` type, it is not
+used or produced by the following functions.
+
+Trino supports three functions for querying JSON data:
```

Comment:
Empty line before and after lists or make it a sentence

Reply:
It is a sentence. Sometimes the formatting might be misleading due to wrapping lines at 80 characters.